src/rickbox/database/models.py

```diff
@@ -0,0 +1,60 @@
+# TODO: Add methods to classes for interfacing with the database
```

Comment:
Are these models currently necessary? If not they should be removed from the PR and added in further PRs if or when they are needed.